client/cmdhf14a.c

```diff
@@ -420,6 +422,13 @@ int CmdHF14AReader(const char *Cmd)
 	c.arg[2] = 0;
 	SendCommand(&c);
 
+	if (isMifareClassic) {		
+		if ( detect_classic_prng() )
+			PrintAndLog("Prng detection: WEAK (darkside)");
+		else
+			PrintAndLog("Prng detection: HARDEND (hardnested)");		
```

Comment:
"[hardened](https://en.wiktionary.org/wiki/hardened)"

This is in a few places throughout your changes.